src/Engine.sol

```diff
@@ -42,6 +52,11 @@ contract Engine is IEngine, EIP712, EIP7412, MulticallablePayable {
                                IMMUTABLES
     //////////////////////////////////////////////////////////////*/
 
+    /// @notice Kwenta owned/operated multisig address that can authorize upgrades
+    /// @dev if this address is the zero address, then the Engine will no longer be upgradeable
+    /// @dev making immutable because the pDAO address will *never* change
+    address internal immutable pDAO;
```

Comment:
**Question:** Sure we don't want to make the contract ownable with the owner in state so that we can change the owner address if needed?